
FAIL history.back() fires the navigate event when reversing a pushState() assert_equals: expected "traverse" but got "push"

